### ISSUE TITLE
Do not show "x files are lint free" message when gem error occurs

### ISSUE THREAD

**Issue by FarSeeing** [ahmednuaman/grunt-scss-lint#32]
Up to now when gem ends running with some error the "x files are lint free" string is also reported:

```
Running "scsslint:all" (scsslint) task
Running scss-lint on all
>> scss-lint failed with error code: 78
>> and the following message:Error: Command failed:
>> 2 files are lint free
Results have been written to: scss-lint-report.xml
```
